src/app/static/src/app/components/plots/MapLegend.vue

```diff
@@ -12,7 +12,7 @@
                 <label>{{indicatorMetadata.name}}</label>
                 <div class="legend" v-for="(level, index) in scaleLevels" v-bind:key="index">
                     <i v-bind:style="level.style"></i>
-                    <span class="level">{{ level.val }}</span>
+                    <span class="level">{{ level.label }}</span>
```

Comment:
Why this level change?

Reply:
No actual change to what is shown here. Previously we were just returning without an explicit type, but I have added one and thought "label" was a better description of what this actually is over "val". See https://github.com/mrc-ide/hint/pull/944/files#diff-74523534d090d132b38de56ec6d1076c4c901addb21445974859b8494fff774fL51